back/lib/shared/neon/neon.go

```diff
@@ -0,0 +1,52 @@
+package neon
+
+import (
+  "database/sql"
+  "fmt"
+  "log"
+  "github.com/KrispyTech/airneis/config"
+  
+  _ "github.com/lib/pq"
+)
+
+func CheckVersion() {
+  config, err := config.InitializeConfig()
+	if err != nil {
+		log.Fatal("Unable to load config - ", err.Error())
+	}
+  username, err := config.Handler.VaultClient.ReadSecret("neon_username")
+	if err != nil {
+		println(err)
+  }
+  secret, err := config.Handler.VaultClient.ReadSecret("neon_secret")
+	if err != nil {
+		println(err)
+	}
+  server, err := config.Handler.VaultClient.ReadSecret("neon_server")
+	if err != nil {
+		println(err)
+	}
+
+  connStr := "postgresql://" + username + ":" + secret + "@" + server + "/airneis?sslmode=require"
```

Comment:
Instead of using "+ " inside a string, in Go, there is a better practice in using [string formating](https://gobyexample.com/string-formatting). You can use something like this where `%s` would contain the string of a variable: 

```suggestion
connStr := fmt.Sprintf("postgresql://%s:%s@%s/airneis?sslmode=require", username, secret, server)
```